ui/util/lbrytv.js

```diff
@@ -9,5 +9,10 @@ function generateEmbedUrl(claimName, claimId) {
   return `${URL}/$/embed/${claimName}/${claimId}`;
 }
 
+function generateDownloadUrl(claimName, claimId, apiUrl) {
+  const streamUrl = generateStreamUrl(claimName, claimId, apiUrl);
```

Comment:
Can these also get wrapped similar to embed links for posterity/readability? E.g. lbry.tv/$/download/${claimName}/${claimId}

Reply:
Yeah. Good idea